client/resource_group/controller/controller.go

```diff
@@ -356,22 +357,27 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 						if err = proto.Unmarshal(item.Kv.Value, group); err != nil {
 							continue
 						}
-						if item, ok := c.groupsController.Load(group.Name); ok {
-							gc := item.(*groupCostController)
+						if gc, ok := c.loadGroupController(group.Name); ok {
```

Comment:
Should we reset the tombstone flag in here if the user deletes it and creates it again?

Reply:
Nice catch. Fixed.